### PR TITLE
Setup classes

### PR DESCRIPTION
**Pull request by David-Lanzz**
# Hello Reviewer!!!
200w_s

## I have completed the assigned tasks to me and below is a summary of my work
- Correct use of Git flow
- No linter errors
- Creation of Teacher, Student, and Person classes
- Inheritance and Polymorphism implemented
- Task completion
## If you have any issues or suggestions, please let me know via the comment section

### CODE REVIEW

**Review by OyePriscilla — COMMENTED** [review]
Hi @David-Lanzz 👍, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that the tech stack and future feature of the README file has N/A as it content. Kindly remove those sections that are not applicable to your project and update the future feature with at least one feature. Thanks. 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,132 @@
-# OOP-School-Library
\ No newline at end of file
+<div align="center">
+  <h1><b>DECODE MORSE MESSAGE</b></h1>
+</div>
```
Comment (emmanuellekamwa) [inline README.md:3]: - [ ] The title of your README does not match the milestone you are working on. Please kindly change that.

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED ❌⛔🛑**
Hello @David-Lanzz 👋
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### Highlights
- Descriptive PR.
- Good commit messages.

## Required Changes ♻️
_Check the comments under the review._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @David-Lanzz 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

  unnamed


## Highlights
- No linter error :heavy_check_mark: 
 
## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**